CONTRIBUTING.md

```diff
@@ -6,16 +6,10 @@ conduct themselves in a professional and respectful manner.
 
 ## Peer review
 
-Although this is an open source project, a review is required from one of the following committers prior to merging a
-Pull Request:
+Although this is an open source project, an approval is required from at least two of the
+[CNF Cert organization members](https://github.com/orgs/test-network-function/people) prior to merging a Pull Request.
```

Comment:
As I understand it, this is the group with approve rights. Is this correct?

Reply:
You need at least two from people with "write/maintain" privileges. anyone can approve a PR, but the only ones that counts are from maintainers of the project.

Reply:
Use this list: https://github.com/orgs/test-network-function/teams/cnfcert/members.

Reply:
Thanks! I've updated the commit.